File: year2/mpp/java/MotorcycleClientServer/MotorcycleCommon/src/main/java/dtos/ParticipantDto.java
package dtos;

import domain.Participant;
import domain.Race;

import java.io.Serializable;

public class ParticipantDto implements Serializable {
    private static final long serialVersionUID = 7331115341259248461L;
    private Race race;
    private Participant participant;

    public ParticipantDto(Race race, Participant participant) {
        this.race = race;
        this.participant = participant;
    }

    public Race getRace() {
        return race;
    }

    public void setRace(Race race) {
        this.race = race;
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    @Override
    public String toString() {
        return "Name: " + participant.getName() + ", Race: " + race.getCapacity() + "cc";
    }
}

File: year2/mpp/c#/MotorcycleClientServer/MotorcycleCommon/Dtos/ParticipantDto.cs
using System;

using MotorcycleContest.Domain.Entities;

namespace MotorcycleContest.Domain.Dtos
{
    [Serializable]
    public class ParticipantDto
    {
        public Race Race { get; set; }
        public Participant Participant { get; set; }

        public ParticipantDto(Race race, Participant participant)
        {
            this.Race = race;
            this.Participant = participant;
        }

        public override string ToString()
        {
            return "Name: " + Participant.Name + ", Race: " + Race.Capacity;
        }
    }
}
